CDS-57217-account level support for srv and env (#1908)
harness/developer-hub: docs/continuous-delivery/gitops/harness-git-ops-basics.md

## Service

A Harness GitOps service is the same as any other Harness service.

A service represents your microservices and other workloads logically. A Service is a logical entity to be deployed, monitored, or changed independently.

> You can create or select services from an account, org, and project level. For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).

Author: ```suggestion
You can create or select services from an account, org, or project level. For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).
```

## Service instance

Service instances represent the dynamic instantiation of a service you sync via Harness GitOps.

For example, for a service representing a Docker image, service instances are the number of pods running with the Docker image.

A single service can have multiple service instances. For example, one for Dev, one for QA, and one for Prod.

## Environments

Harness environments represent your live environment logically (QA, Prod, etc). In Harness GitOps, an environment is the live state of the infrastructure.

> You can create or select environments from an account, org, and project level. For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).

Author: ```suggestion
You can create or select environments from an account, org, or project level. For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).
```

## Service

A Harness GitOps service is the same as any other Harness service.

A service represents your microservices and other workloads logically. A Service is a logical entity to be deployed, monitored, or changed independently.

> You can create or select services from an account, org, and project level. Account level services need account level connectors. Similarly, org and project level services need org and project level connectors respectively.

Author: ```suggestion
You can create or select services from an account, organization, and project level. 

An account level service can only reference connectors for the manifests and artifacts within the account. These services are global and cannot have dependencies at a lower hierarchy level.
```

For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).

## Service instance

Service instances represent the dynamic instantiation of a service you sync via Harness GitOps.

For example, for a service representing a Docker image, service instances are the number of pods running with the Docker image.

A single service can have multiple service instances. For example, one for Dev, one for QA, and one for Prod.

## Environments

Harness environments represent your live environment logically (QA, Prod, etc). In Harness GitOps, an environment is the live state of the infrastructure.

> You can create or select environments from an account, org, and project level. Account level environments need account level connectors. Similarly, org and project level environments need org and project level connectors respectively.

Author: ```suggestion
You can create or select environments from an account, organization, and project level. 

An account level environments can only reference connectors within the account. They cannot have dependencies at a lower hierarchy level.
```

## Service

A Harness GitOps service is the same as any other Harness service.

A service represents your microservices and other workloads logically. A Service is a logical entity to be deployed, monitored, or changed independently.

> You can create or select services from an account, organization, and project level. 

Author: ```suggestion
You can create or select services from an account, organization, or project level. 
```

An account level service can only reference connectors for the manifests and artifacts within the account. These services are global and cannot have dependencies at a lower hierarchy level.

For more information, go to [Services and environments overview](/docs/continuous-delivery/get-started/services-and-environments-overview).

## Service instance

Service instances represent the dynamic instantiation of a service you sync via Harness GitOps.

For example, for a service representing a Docker image, service instances are the number of pods running with the Docker image.

A single service can have multiple service instances. For example, one for Dev, one for QA, and one for Prod.

## Environments

Harness environments represent your live environment logically (QA, Prod, etc). In Harness GitOps, an environment is the live state of the infrastructure.

> You can create or select environments from an account, organization, and project level. 

Author: ```suggestion
You can create or select environments from an account, organization, or project level. 
```

> An account level environments can only reference connectors within the account. They cannot have dependencies at a lower hierarchy level.

Author: ```suggestion
An account level environment can only reference connectors within the account, and cannot have dependencies at a lower hierarchy level.
```
